packages/nimble-components/src/combobox/index.ts

```diff
@@ -70,6 +70,26 @@ export class Combobox
     @observable
     public controlWrapper?: HTMLElement;
 
+    public override get value(): string {
+        return super.value;
+    }
+
+    // This override is to work around an issue in FAST where an old filter value
+    // is used after programmatically setting the value property.
+    // See: https://github.com/microsoft/fast/issues/6749
+    public override set value(next: string) {
+        super.value = next;
+        if (!this.valueUpdatedByInput) {
+            // Using index notation to avoid error from accessing private member
+            // eslint-disable-next-line @typescript-eslint/dot-notation
+            this['filter'] = '';
```

Comment:
That is what I was doing initially, but I thought that setting a value was not like incrementally typing, but more like picking from the list. Of course, that's only true if you set a value that is in the list. Still, I think the point of setting the filter is to *help you make your choice* interactively, whereas setting a value is really committing a final value. For that reason, it seems like clearing the filter is better than setting it to the value. Is that convincing? :)

Reply:
The oddity I think we have to reconcile is that if a user types "fo" (let's say `autocomplete` is set to `list`), with the options "foo" and "bar", user will see "foo" in the dropdown. If, instead of ever typing anything, the user set the value to "fo", and then opened the dropdown, they will see "foo" and "bar" in the dropdown, which could be considered strange, since there is no visual distinction between the two scenarios. That said, the behavior you have here is in line with my preference as I outlined in [the issue I filed to FAST](https://github.com/microsoft/fast/issues/6749). So I, personally, don't have a problem with the behavior you've created, I'm just vaguely concerned it goes against FAST's expectations for behaviors.